deepchecks/utils/distribution/plot.py

```diff
@@ -362,3 +362,27 @@ def sort_int_and_strings(a, b):
     )
 
     return traces, yaxis_layout
+
+def word_counts_bar_traces(
```

Comment:
What's the difference between this and the regular categorical drift plot?

Reply:
@noamzbr the regular categorical accepts a categorical series and calculates the counts and the train-test alignment as well, here the data is already aligned and in frequencies

Reply:
So maybe let's put this plot in the drift.py file? Not a must, only if you think we may use this elsewhere